apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/PartDEventTransformerV2Test.java

```diff
@@ -872,6 +876,19 @@ public void shouldHavePayment() throws Exception {
     assertTrue(compare.equalsDeep(eob.getPayment()));
   }
 
+  @Test
```

Comment:
I know the rest of the class is un-javadoc'ed but we are moving in that direction so please add one to this new test